README.md

```diff
@@ -111,9 +126,12 @@ end
 ```
 
 Your answer:
-```text
+```
+describe and context are typically the same in the sense that there's no functional difference but how they are used allows the tests to be more understandable. The purpose of describe is to wrap a set of tests against one functionality. It's usually at the top level. I like to think of it as the introduction to a text like its name or description.
 
-Replace this with your answer
+context wraps a set of tests  under describe that wraps a text for methods under the same state. I think of this as a unique instance to the top level test. For example,
+
+you will DESCRIBE "greeting" as the title and the context is how you will greet if in english specifically. It's still related to greeting but it's a specific context of the greeting that needs to be tested.
```

Comment:
i think this is right, but I'd be careful about saying describe and context are actually the same. they have different purposes which you mention right after